index.html

```diff
@@ -138,11 +138,12 @@ <h4 class="title">Skills</h4>
 			<h2>Contact me</h2>
 			<p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
 		</div>
-		<form action="https://formspree.io/f/moqbvjoj" method="POST" class="contact-form">
+		<form action="https://formspree.io/f/moqbvjoj" method="POST" class="contact-form" novalidate>
 			<input type="text" placeholder="Your name" maxlength="30" required>
 			<input type="email" name="email" id="email" placeholder="Your email" required>
 			<textarea name="text" id="text" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
 			<button type="submit" class="button btn">Get In Touch</button>
+			<small></small>
 		</form>
```

Comment:
- [ **OPTIONAL** ] Good job doing the change required by the previous reviewer. But I think it would be better if you display your **Error Message** between the `textarea` field and the `Get In Touch` button. This will help users if they didn't scroll the whole page up to the bottom to correctly see the error message.